packages/svelte/src/lib/components/ViewportPortal/ViewportPortal.svelte

```diff
@@ -1,10 +1,10 @@
 <script lang="ts">
-  import { portal } from '$lib/actions/portal';
+  import { hideDuringSSR, portal } from '$lib/actions/portal';
   import type { ViewportPortalProps } from './types';
 
   let { target = 'front', children, ...rest }: ViewportPortalProps = $props();
 </script>
 
-<div use:portal={`viewport-${target}`} {...rest}>
+<div use:portal={`viewport-${target}`} style:display={hideDuringSSR().display} {...rest}>
```

Comment:
the naming here is a bit confusing to me:

`style:display={hideDuringSSR().display`

is this hidden during SSR? why is there a `display` field?